composer.json

```diff
@@ -72,7 +73,8 @@
         "analyze": "psalm"
     },
     "suggest": {
-        "spiral/roadrunner-cli": "Provides RoadRunner installation and management CLI tools"
+        "spiral/roadrunner-cli": "Provides RoadRunner installation and management CLI tools",
+        "ext-protobuf": "Provides Protocol Buffers support"
```

Comment:
It would be beneficial to include a recommendation about this extension in the documentation. It's important to emphasize that it impacts performance. Without this extension, JSON operates faster than Proto.